transform/reverse/reverse_test.go

```diff
@@ -0,0 +1 @@
+package reverse
```

Comment:
Either add tests or delete this?

Reply:
I'll add tests. That was on my todo list thanks for catching this and reminding me.

Reply:
Just added a test. Done!

Reply:
Wait no. Test failed. That's what I get for using GitHub copilot. Was just excited that it wrote the test for me 😅

Reply:
Should this just be in a general utils package?

Reply:
I'm jealous, extremely impatiently waiting on copilot access.

![image](https://user-images.githubusercontent.com/1220444/124057500-6e360a80-d9dc-11eb-8319-714a51c5f84e.png)

Reply:
> Should this just be in a general utils package?

Go frowns upon this practice a lot and I tend to agree. Easy to dump a bunch of unrelated stuff in there. I do think the naming for this should be a lot smarter cause it turns out that co-pilot actually wrote a test for a function that returns a complement which without the package name is pretty much what `reverse.Complement` looks like.

Reply:
ok, that's fair! If it's the go way, might as well

Reply:
Also tricky thing about `reverse.complement` is that it's used almost everywhere and was causing serious cyclic dependency issues to the point where it kind of needed its own package that didn't rely on anything else. This is probably the case with a lot of util like functions.

Reply:
Just fixed this in latest push. I moved `ReverseComplement` to the `transform` package so now you call it with `transform.ReverseComplement`. I also pushed `Reverse` and `Complement` functions and Example tests. @Koeng101 @jkhales